conda-store-server/LICENSE

```diff
@@ -7,11 +10,11 @@ met:
 1. Redistributions of source code must retain the above copyright
 notice, this list of conditions and the following disclaimer.
 
-2. Redistributions in binary form must reproduce the above copyright
+1. Redistributions in binary form must reproduce the above copyright
 notice, this list of conditions and the following disclaimer in the
 documentation and/or other materials provided with the distribution.
 
-3. Neither the name of the copyright holder nor the names of its
+1. Neither the name of the copyright holder nor the names of its
```

Comment:
Same note as for the `LICENSE` file.

Reply:
Yep I copy-pasted across the files so that they were the same